ui/package.json

```diff
@@ -98,7 +98,7 @@
     "@types/ember__utils": "^4.0.2",
     "@types/qunit": "^2.19.4",
     "@types/rsvp": "^4.0.4",
-    "@types/shell-quote": "^1.7.1",
+    "@types/shell-quote": "^1.7.3",
```

Comment:
The type is out of date with our actual version of `shell-type` -- should they be the same? Odd that we're getting an audit alert for a types version 🤔

Reply:
Oh, this might actually be an issue in our yarn lock. I see two shell-quotes. (our 1.8.1 and an old 1.6.1). Stay tuned, let me re do this and see if I can figure out why that is happening.

Reply:
`yarn why <package>` might be helpful for you there!